ide_profiles/xcode/README.md

```diff
@@ -1,7 +1,12 @@
 ## Xcode IDE profile
 
-Use this profile to edit and build the project using Xcode.
+Use this profile to edit the project using Xcode.
 
 To use this profile, open `ExtendedKF.xcodeproj` in this folder.
 
-To generate an Xcode project or update existing one, use `cmake -G "Xcode" ../..` command in the current directory.
+To build using Xcode, re-generate this project as follows:
+
```

Comment:
It looks like when I remove empty line, markdown displays the document incorrectly. All bullet points are in one line. I don't think we should remove it.